docs/gitbook/architecture/public-private-control-model.md

```diff
@@ -0,0 +1,50 @@
+# Public/Private/Control Architectural Model
+
+The Interbit **Public/Private/Control (PPC)** architectural model is intended as a template for building Interbit applications that respect user's privacy.
+
+The PPC model is used within the [Interbit Accounts](https://accounts.interbit.io) so that each user will have their own private chain to ensure privacy of their personal data. It is also implemented in the *template* to enable developers to implement PPC applications that ensure user data privacy.
+
+The PPC model also enables *Chain Authorization* where users can authorize secure data sharing of their personal data in their private accounts chain with their private application chains.
+
+### Chain Roles within the PPC Model
+
+The PPC model defines specific roles for Public, Private and Control chains, but can be expanded to include chains for sharing data between users. The responsibilities of the specific chains in the PPC model are:
+
+**Public Chain**: Every user can load the public chain. The public chain contains the configuration required to create new private chains when a user launches the application.
+
+**Private Chain**: Each user has their own private chain for privacy, and only the owner can load and interact with that chain. The ACL of the private chain will be restricted, typically containing the public key(s) of the chain's owner and the chain ID of the control chain that created it.
+
+**Control Chain**: This is the administrative chain that manages the creation of private chains and is the source of truth of system data on the public chain (shared via a read join). Only administrators have access to this chain. Because this chain contain information on the private chains that it creates (join configuration or block history), it is a potential source of metadata leakage and should not be loaded on users browsers.
```

Comment:
I don't think it's fair to say only administrators have access to this. Anyone can have access but there are serious implications on the data leaked and shared and it should only be done after careful consideration. 

There are some cases where the metadata leakage is desired, as in a sharing model where we want everyone to know when a user was created and who they are interacting with in a pseudonymous environment using user chain IDs, but don't want to share their personal data.